### PR TITLE
Feat/update style popup map

### PR DESCRIPTION
**Pull request by diego-morales-flores-1996**
## Pull request checklist

Please check if your PR fulfills the following requirements:
- [ ] Make sure you are requesting to pull a topic/feature/bugfix branch (right side). Don't request your master!
- [ ] Make sure you are making a pull request against the develop branch (left side). Also you should start your branch off our develop.
- [ ] Check the commit's or even all commits' message styles matches our requested structure.
- [ ] Check your code additions will fail neither code linting checks nor unit test.

## Pull request type

 

 

Please check the type of change your PR introduces:
- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe): 

## What is the current behavior?

Issue Number: N/A

## What is the new behavior?

-
-
-

## Does this introduce a breaking change?

- [ ] Yes
- [ ] No

### CODE REVIEW

File: src/components/elements/ImageGallery/ImagePreview.tsx [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+const ImageGalleryPreviewer: FC<ImageGalleryPreviewerProps> = ({
+  data,
+  onCLose,
+  className,
+  WrapperClassName,
+  backdropClassName,
+  ...rest
+}) => {
+  return ReactDOM.createPortal(
```
Comment (roguenet) [inline src/components/elements/ImageGallery/ImagePreview.tsx:27]: Please avoid the `return` statement for single statement arrow functions
```
}) => ReactDOM.createPortal(
```

File: src/components/elements/ImageGallery/ImagePreview.tsx [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+              width: 32,
+              height: 32,
+              className: "fill-neutral-1000"
+            }}
+            onClick={onCLose}
+          />
+        </div>
+
+        {/* Preview */}
+        <ImageWithPlaceholder imageUrl={data.fullImageUrl} alt={data.label || ""} />
```
Comment (roguenet) [inline src/components/elements/ImageGallery/ImagePreview.tsx:52]: `??`

Comment (diego-morales-flores-1996) [reply src/components/elements/ImageGallery/ImagePreview.tsx]: done

File: src/components/elements/ImageGallery/ImagePreview.tsx (hunk 1 above)
Comment (diego-morales-flores-1996) [reply src/components/elements/ImageGallery/ImagePreview.tsx]: done

**Review by roguenet — APPROVED** [review]

**Review by dottyy — APPROVED** [review]

**Pull request merged**